test/motor_message_test.cc

```diff
@@ -99,11 +99,8 @@ TEST(MotorMessageTest, motor_message_register_values) {
     mc.setRegister(static_cast<MotorMessage::Registers>(0x01));
     ASSERT_EQ(MotorMessage::REG_BRAKE_STOP, mc.getRegister());
 
-    mc.setRegister(static_cast<MotorMessage::Registers>(0x10));
-    ASSERT_EQ(MotorMessage::REG_ERROR_COUNT, mc.getRegister());
-
-    mc.setRegister(static_cast<MotorMessage::Registers>(0x0B));
```

Comment:
I did not realize these were here.   There have been numerous old register repurposing because the registers do not exist in our main.c usage.
So sorry, I missed these.  This comment applies to most all things here.